Gemfile

```diff
@@ -2,8 +2,9 @@ source 'https://rubygems.org'
 
 gem 'rails', '~> 4.1.6'
 gem 'envied', '~> 0.8.1'
-gem 'pg', '~> 0.17.1'
+gem 'pg', '~> 0.18.0'
 gem 'virtus', '~> 1.0.5'
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw]
```

Comment:
This gem is required for rails to work on Windows (it doesn't have system zoneinfo dir). More info: https://github.com/tzinfo/tzinfo/wiki/Resolving-TZInfo::DataSourceNotFound-Errors